src/js/app.js

```diff
@@ -1,5 +1,59 @@
 import $ from "jquery";
+import 'slick-carousel';
 
 $(function() {
   console.log('環境構築完了');
 })
+
+$(function() {
+  //ハンバーガーボタン⇨メニュースクロール
+  $('.l-header__hamburger').on('click', function() {
```

Comment:
[imo]

`$function()` は複数必要ないので以下のように「環境構築完了」の文字は消して、その中に全ての js の処理を記述していってもらっても良いですか？

```suggestion
$(function() {
  //ハンバーガーボタン⇨メニュースクロール
  $('.l-header__hamburger').on('click', function() {
  ・・・
})
```